src/System.Numerics.Vectors/tests/System.Numerics.Vectors.Tests.csproj

```diff
@@ -35,6 +35,13 @@
       <Link>System\MathF.netstandard.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
```

Comment:
Do we want these tests on `uap` as well?

Reply:
Just to check... so we don't require `uapaot` here as well? Please excuse my ignorance :-(

between, will do the same changes to the performance test project as well... they are not there right now.

Reply:
Right. TargetGroup can only have values listed in Configurations.props.

Reply:
Thanks @jkotas! Now I understand how that ties up!